spec/post_spec.rb

```diff
@@ -13,15 +19,16 @@
     expect(post).to_not be_valid
   end
 
-  it 'is not valid with a negative comments_counter' do
+  it 'is not valid with a negative likes_counter' do
     user = User.create(name: 'Bob Smith', posts_counter: 1)
-    post = user.posts.new(title: 'Sample Post', comments_counter: -1, likes_counter: 0)
-    expect(post).to_not be_valid
+    post = user.posts.new(title: 'Sample Post', comments_counter: 1, likes_counter: -1)
+    expect(post).not_to be_valid
+    expect(post.errors[:likes_counter]).to include('must be greater than or equal to 0')
   end
 
-  it 'is valid with zero comments_counter' do
+  it 'is valid with zero likes_counter' do
     user = User.create(name: 'Eve Johnson', posts_counter: 4)
-    post = user.posts.new(title: 'Sample Post', comments_counter: 0, likes_counter: 2)
+    post = user.posts.create(title: 'Sample Post', comments_counter: 3, likes_counter: 0)
     expect(post).to be_valid
   end
 end
```

Comment:
- [ ] Great work on making your test pass but i see that previous reviewer asked you to add some more tests for the likes counter but you have modified the comments_counter tests for likes_counter 😨 Now you don't have any test to verify the comments_counter 😮 Kindly make separate tests for the comments_counter and separate for likes_counter to follow the project requirements 👍